lib/fpm/package/virtualenv.rb

```diff
@@ -135,7 +139,15 @@ def input(package)
       target_args << package
     end
 
-    pip_args = [python_exe, pip_exe, "install", "-i", attributes[:virtualenv_pypi]] << extra_index_url_args << find_links_url_args << target_args
+    # Installing from local project path has an advantage to now having to upload
+    # python package to pypi server just to make distro package.
+    if attributes[:virtualenv_local_project_path]
```

Comment:
you could remove the `:default => ""` which would imply the default is nil, making this check `if ...` valid because `nil` is falsey in Ruby.

Thoughts?

Either a falsey check (with no default (nil)) or `.empty?` is fine with me.